serenata_toolbox/datasets/__init__.py

```diff
@@ -52,7 +53,8 @@ def __init__(self, local_directory=None):
         self.downloader = Downloader(
             local_directory,
             bucket=self.remote.bucket,
-            **self.remote.credentials
+            **self.remote.credentials,
+            timeout=timeout
```

Comment:
Would you mind passing all specified keyword arguments before unpacking the others? That is to say:

```python
Downloader(local_directory, bucket=self.remote.bucket, timeout=None,  **self.remote.credentials)
```

instead of

```python
Downloader(local_directory, bucket=self.remote.bucket, **self.remote.credentials, timeout=None)
```